external/loaders/loaders/batches/_batch.py

```diff
@@ -66,8 +60,6 @@ def batches_from_geodata(
         timesteps_per_batch (int, optional): Defaults to 1.
         random_seed (int, optional): Defaults to 0.
         res: grid resolution, format as f'c{number cells in tile}'
-        subsample_size: draw a random subsample from the batch of the
-            specified size along the sampling dimension
```

Comment:
Is it necessary to transfer this feature into the training code, or is this feature unused?

Reply:
This is unused, I checked with Andre who was the only past user and he said it was fine to remove.